packages/block-library/src/social-link/index.php

```diff
@@ -19,7 +19,7 @@ function render_block_core_social_link( $attributes, $content, $block ) {
 
 	$service    = ( isset( $attributes['service'] ) ) ? $attributes['service'] : 'Icon';
 	$url        = ( isset( $attributes['url'] ) ) ? $attributes['url'] : false;
-	$label      = ( isset( $attributes['label'] ) ) ? $attributes['label'] : block_core_social_link_get_name( $service );
+	$label      = ( isset( $attributes['label'] ) ) ? $attributes['label'] : block_core_social_link_get_name( $service ) . ": " . $url;
```

Comment:
@swissspidy did you have any thoughts on this one? It's really simple, but I'm not sure if this string building would be an issue.

Reply:
Will concatenating strings like that work OK for RTL languages? Maybe we should do something like this instead?
```suggestion
	$label      = ( isset( $attributes['label'] ) ) ? $attributes['label'] : sprintf(
		/* translators: %1$s: Social-network name. %2$s: URL. */
		__( '%1$s: %2$s', 'gutenberg' ),
		block_core_social_link_get_name( $service ),
		$url
	);
```